kernel/src/process.rs

```diff
@@ -704,7 +704,7 @@ impl<C: Chip> ProcessType for Process<'a, C> {
                 unallocated_memory_start,
                 unallocated_memory_size,
                 min_region_size,
-                mpu::Permissions::ReadWriteExecute,
+                mpu::Permissions::ReadWriteOnly,
```

Comment:
I think we use `allocate_region` is used to allocate MPU region for flash memory. Shouldn't this be `mpu::ReadExecuteOnly`?

Reply:
`add_mpu_region` is used only for IPC regions. The flash region is allocated on line 1163 as `ReadExecuteOnly` (which hasn't changed).

Reply:
That's right. Sorry, I forgot about IPC use-case. Thanks for clarifying.